docs/en-US/component/input-number.md

```diff
@@ -79,38 +79,41 @@ input-number/controlled
-| blur   | remove focus the input component | —          |
+## API
+
+### Attributes
+
+| Name                                         | Description                                      | Type                                          | Default   |
+| -------------------------------------------- | ------------------------------------------------ | --------------------------------------------- | --------- |
+| model-value / v-model                        | binding value                                    | ^[number]                                     | —         |
+| min                                          | the minimum allowed value                        | ^[number]                                     | -Infinity |
+| max                                          | the maximum allowed value                        | ^[number]                                     | Infinity  |
+| step                                         | incremental step                                 | ^[number]                                     | 1         |
+| step-strictly                                | whether input value can only be multiple of step | ^[boolean]                                    | false     |
+| precision                                    | precision of input value                         | ^[number]                                     | —         |
+| size                                         | size of the component                            | ^[enum]`'large' \| 'default' \| 'small'`      | default   |
+| readonly                                     | same as `readonly` in native input               | ^[boolean]                                    | false     |
+| disabled                                     | whether the component is disabled                | ^[boolean]                                    | false     |
+| controls                                     | whether to enable the control buttons            | ^[boolean]                                    | true      |
+| controls-position                            | position of the control buttons                  | ^[enum]`'' \| 'right'`                        | —         |
+| name                                         | same as `name` in native input                   | ^[string]                                     | —         |
+| label                                        | same as `label` in native input                  | ^[string]                                     | —         |
+| placeholder                                  | same as `placeholder` in native input            | ^[string]                                     | —         |
+| id                                           | same as `id` in native input                     | ^[string]                                     | —         |
+| value-on-clear<VersionTag version="2.2.0" /> | value should be set when input box is cleared    | ^[number] / ^[null] / ^[enum]`'min' \| 'max'` | —         |
```

Comment:
@chenxch maybe we should keep the old way of this since this will produce error in Chinese translations.

Reply:
Overall it LGTM.

Reply:
The use of `<...>` in table is easy to cause format errors after translation. I think a custom Markdown plugin can be added to deal with this problem.

Same like
 
```
^[enum]`'min' \| 'max'`
```

Reply:
@jw-foss What do you think?

Use as follows

```
^(2.2.0)

^(a11y)

^(deprecated)

//...
```

Reply:
@tolking agreed, maybe we should just apply this way. Do you have the plugin implementation right now?

Reply:
I can write a plugin to implement it.